### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by Anhelbags** [johngodley/redirection#661]
```
Plugin: 3.0.1
WordPress: 4.7.9
PHP: 5.6.33
Browser: Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36

Error: Неверный параметр: per_page
Action: redirection/v1/redirect?orderby=id&per_page=2 GET
Code: 400 Bad Request
Raw: {"code":"rest_invalid_param","message":"\u041d\u0435\u0432\u0435\u0440\u043d\u044b\u0439 \u043f\u0430\u0440\u0430\u043c\u0435\u0442\u0440: per_page","data":{"status":400,"params":{"per_page":"per_page \u0434\u043e\u043b\u0436\u0435\u043d \u0431\u044b\u0442\u044c \u043c\u0435\u0436\u0434\u0443 5 (\u0432\u043a\u043b\u044e\u0447\u0438\u0442\u0435\u043b\u044c\u043d\u043e) \u0438 200 (\u0432\u043a\u043b\u044e\u0447\u0438\u0442\u0435\u043b\u044c\u043d\u043e)"}}}

Error: Неверный параметр: per_page
Action: redirection/v1/group?per_page=2 GET
Code: 400 Bad Request
Raw: {"code":"rest_invalid_param","message":"\u041d\u0435\u0432\u0435\u0440\u043d\u044b\u0439 \u043f\u0430\u0440\u0430\u043c\u0435\u0442\u0440: per_page","data":{"status":400,"params":{"per_page":"per_page \u0434\u043e\u043b\u0436\u0435\u043d \u0431\u044b\u0442\u044c \u043c\u0435\u0436\u0434\u0443 5 (\u0432\u043a\u043b\u044e\u0447\u0438\u0442\u0435\u043b\u044c\u043d\u043e) \u0438 200 (\u0432\u043a\u043b\u044e\u0447\u0438\u0442\u0435\u043b\u044c\u043d\u043e)"}}}
```

**Comment by Anhelbags** [comment]
error never leaves the page. how to remove the error?

**Comment by johngodley** [comment]
Click on 'screen options' and change the per page limit to 25

**Comment by johngodley** [comment]
You can also use the 3.1 beta:

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by johngodley** [comment]
I've now released version 3.1. I will close this ticket for now, but please reopen if the problem persists
